scripts/generate-app-config/generateRouteConfig.js

```diff
@@ -39,15 +40,32 @@ const routeItem = (routePath, { contentPath, name }, props, routeImporter) => ({
   },
 });
 
+const evidenceProps = (contentConfig, routeImporter) => {
+  const contentCopy = Object.assign({}, contentConfig);
+  return Object.keys(contentCopy).reduce((acc, viewportKey) => {
+    const viewport = contentCopy[viewportKey];
+    const contentPath = routeImporter.addImport(ImportAggregator.relativePath(viewport));
+    acc.push({
+      viewport: viewportKey,
+      contentPath,
+    });
+    return acc;
+  }, []);
+};
+
 /**
  * Sets up content route item. All content items are wrapped with the content wrapper.
  */
 const contentRouteItem = (routePath, { contentPath, name, identifier }, props, type, routeImporter) => {
-  const relativeContent = routeImporter.addImport(ImportAggregator.relativePath(contentPath), name, identifier);
-  let contentProps = {
-    content: relativeContent,
-    props,
-  };
+  let relativeContent;
+  let contentProps;
+  if (typeof contentPath === 'string') {
```

Comment:
What are the content path options that are passed down?

Reply:
Single string content paths, or a hash of path strings.